base-cert-manager/certificates.yaml

```diff
@@ -0,0 +1,56 @@
+# https://cert-manager.io/docs/configuration/selfsigned/#bootstrapping-ca-issuers
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: cockroachdb-selfsigned-issuer
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: cockroachdb-ca
+spec:
+  isCA: true
+  commonName: cockroachdb-ca
+  dnsNames:
+    - cockroachdb-ca
+  secretName: cockroachdb-ca
+  issuerRef:
+    name: cockroachdb-selfsigned-issuer
+    kind: Issuer
+    group: cert-manager.io
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: cockroachdb-ca-issuer
+spec:
+  ca:
+    secretName: cockroachdb-ca
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: node
+spec:
+  issuerRef:
+    name: cockroachdb-ca-issuer
+    kind: Issuer
+  commonName: node
+  dnsNames:
```

Comment:
Cockroach docs suggest adding 
```
X509v3 Extended Key Usage:
                TLS Web Server Authentication, TLS Web Client Authentication
```
So we should add this as per cert-manager docs:
```
usages:
    - server auth
    - client auth
```